src/Eurofurence.App.Infrastructure.EntityFramework/AppDbContext.cs

```diff
@@ -0,0 +1,82 @@
+            : base(options)
+        {
+        }
+
+        public virtual DbSet<AnnouncementRecord> Announcements { get; set; }
+        public virtual DbSet<TableRegistrationRecord> TableRegistrations { get; set; }
+        public virtual DbSet<AgentClosingResultRecord> AgentClosingResults { get; set; }
+        public virtual DbSet<ItemActivityRecord> ItemActivitys { get; set; }
+        public virtual DbSet<PrivateMessageRecord> PrivateMessages { get; set; }
+        public virtual DbSet<DealerRecord> Dealers { get; set; }
+        public virtual DbSet<EventConferenceDayRecord> EventConferenceDays { get; set; }
+        public virtual DbSet<EventConferenceTrackRecord> EventConferenceTracks { get; set; }
+        public virtual DbSet<EventFeedbackRecord> EventFeedbacks { get; set; }
+        public virtual DbSet<EventRecord> Events { get; set; }
+        public virtual DbSet<FursuitParticipationRecord> FursuitParticipations { get; set; }
+        public virtual DbSet<PlayerParticipationRecord> PlayerParticipations { get; set; }
+        public virtual DbSet<CollectionEntryRecord> CollectionEntries { get; set; }
+        public virtual DbSet<TokenRecord> Tokens { get; set; }
+        public virtual DbSet<FursuitBadgeImageRecord> FursuitBadgeImages { get; set; }
+        public virtual DbSet<FursuitBadgeRecord> FursuitBadges { get; set; }
+        public virtual DbSet<ImageContentRecord> ImageContents { get; set; }
+        public virtual DbSet<ImageRecord> Images { get; set; }
+        public virtual DbSet<KnowledgeEntryRecord> KnowledgeEntries { get; set; }
+        public virtual DbSet<KnowledgeGroupRecord> KnowledgeGroups { get; set; }
+        public virtual DbSet<LostAndFoundRecord> LostAndFounds { get; set; }
+        public virtual DbSet<MapEntryRecord> MapEntries { get; set; }
+        public virtual DbSet<MapRecord> Maps { get; set; }
+        public virtual DbSet<PushNotificationChannelRecord> PushNotificationChannels { get; set; }
+        public virtual DbSet<RegSysAccessTokenRecord> RegSysAccessTokens { get; set; }
+        public virtual DbSet<RegSysAlternativePinRecord> RegSysAlternativePins { get; set; }
+        public virtual DbSet<IssueRecord> IssueRecords { get; set; }
+        public virtual DbSet<RegSysIdentityRecord> RegSysIdentities { get; set; }
+        public virtual DbSet<EntityStorageInfoRecord> EntityStorageInfos { get; set; }
+        public virtual DbSet<UserRecord> Users { get; set; }
+        public virtual DbSet<RoleRecord> Roles { get; set; }
+        public virtual DbSet<TopicRecord> Topics { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            base.OnModelCreating(modelBuilder);
+
+            modelBuilder.Entity<RegSysAlternativePinRecord>().Property(x => x.PinConsumptionDatesUtc)
+                .HasConversion(new ValueConverter<List<DateTime>, string>(
+                    v => JsonConvert.SerializeObject(v),
```

Comment:
Can we maybe add either [`Pomelo.EntityFrameworkCore.MySql.Json.Microsoft`](https://www.nuget.org/packages/Pomelo.EntityFrameworkCore.MySql.Json.Microsoft/) or [`Pomelo.EntityFrameworkCore.MySql.Json.Newtonsoft`](https://www.nuget.org/packages/Pomelo.EntityFrameworkCore.MySql.Json.Newtonsoft/) to get native json support with MySQL's json type?
All you would have to do is add one of them, register in `Startup` and call `.HasDataType("json")`

Reply:
Thanks, I implemented it in the latest commit!